packages/fxa-customs-server/test/remote/too_many_password_reset_otp_verifications.js

```diff
@@ -67,6 +67,11 @@ test('/check passwordForgotVerifyOtp by email', async (t) => {
   // the min configurable value is a second
   await new Promise((resolve) => setTimeout(resolve, 1000));
 
+  // this is the third recorded attempt; the third request was rate limited
+  // thus not recorded
+  ip = testUtils.randomIp();
+  await client.postAsync('/check', { ip, email, action });
```

Comment:
I would assert the expected response.

Reply:
Which I expect to the rate limit to be cleared since the retryAfter time has elapsed?

Reply:
Yes.  But this is triggering longer duration limit (the 10 / day limit in actual deployment).